lib/pure/os.nim

```diff
@@ -60,8 +76,9 @@ proc osErrorMsg*(): string {.rtl, extern: "nos$1", deprecated.} =
                           nil, err, 0, addr(msgbuf), 0, nil) != 0'i32:
           result = $msgbuf
           if msgbuf != nil: localFree(msgbuf)
-  if errno != 0'i32:
-    result = $os.strerror(errno)
+  else:
```

Comment:
## On Windows ``GetLastError`` is checked before ``errno``.
  ## Returns "" if no error occurred.

So yes, it's supposed to check both, but it's wrong that result is overwritten with a presumably inferior error message. Either way your change has nothing to do with this PR, so remove it.